app/models/runs_reducers.rb

```diff
@@ -29,16 +20,10 @@ def reduce(subject_id, user_id, extract_ids=[])
       filter[:project_id] = reducible.id
     end
 
-    extract_fetcher = ExtractFetcher.new(filter).including(extract_ids | prior_extracts)
-
+    extract_fetcher = ExtractFetcher.new(filter).including(extract_ids)
```

Comment:
Why not just pass the `extract_ids` when instantiating via the the `ExtractFetcher.new` initializer? The initializer could have a default value and the extract_ids could be an optional param.

The `.including` method only seems to modify the list of extract_ids and return self for chaining / assignment.

Reply:
i think the original idea was that we might need to change this list at several different points in time, but that never really materialized. this is probably a good idea.